functional/tests/core-manage-discovery-jsonrpc.test.mjs

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+import { expect, jest, test, afterAll, beforeAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(30040);
+
+const httpPort = 3336;
+const socketPort = 9009;
+
+beforeAll(async () => {
+  console.log(
+    await utilities.mfState(
+      true,
+      ` -- --manage --discovery --httpPort ${httpPort} --socketPort ${socketPort}`
+    )
+  );
+});
+
+afterAll(async () => {
+  //Stop Mock Firebolt
+  console.log(await utilities.killPort(socketPort));
+});
+
+test(`OPENRPC Response`, async () => {
+  const response = await utilities.fireboltCommand(
+    JSON.stringify({
+      method: "accessibility.closedCaptionsSettings",
```

Comment:
Question: Are we sending firebolt command for manage or discovery method?

Reply:
Changed to -> content.providers